packages/material-renderers/src/layouts/ArrayToolbar.tsx

```diff
@@ -8,20 +8,23 @@ import {
 import AddIcon from '@mui/icons-material/Add';
 import React from 'react';
 import ValidationIcon from '../complex/ValidationIcon';
+import { ArrayTranslations } from '@jsonforms/core/lib/i18n/arrayTranslations';
```

Comment:
This should be imported directly from the jsonforms/core package. If that is not possible, please export the new definitions appropriately.
```suggestion
import { ArrayTranslations } from '@jsonforms/core';
```